__tests__/View-tests/map/mapScreen.test.tsx

```diff
@@ -19,8 +20,12 @@ jest.mock("react-native-maps", () => {
 });
 
 describe("MapScreen UI Tests", () => {
-  const mockNavigation = { goBack: jest.fn() };
+  const mockNavigation = {
+    goBack: jest.fn(),
+    navigate: jest.fn(),
+  };
   const mockFirestoreCtrl = new FirestoreCtrl();
+  const mockDate = new Date();
```

Comment:
When using new Date for a mock make sure to accept any date as an expected argument

Reply:
I get it thank you !